src/bundle/Resources/config/bazinga_js_translation.yml

```diff
@@ -0,0 +1,12 @@
+active_domains:
+    - 'admin_ui_frontend_admin_limitation_pick'
```

Comment:
Do we really need to use `frontend` keyword in the domain name? Looks a bit strange to me. ping @webhdx @Nattfarinn